pkg/actors/actors.go

```diff
@@ -37,6 +37,7 @@ type Actors interface {
 	SaveState(req *SaveStateRequest) error
 	DeleteState(req *DeleteStateRequest) error
 	TransactionalStateOperation(req *TransactionalRequest) error
+	PerformTransaction(req *[]state.TransactionalRequest) error
```

Comment:
Changes in actor layer are not needed for this, please see my review comment about all the suggested changes.